posts/2019/05/spyder-4-coming-closer-beta2-is-now-available.md

```diff
@@ -0,0 +1,148 @@
+
+>A single Language Server can be reused in multiple development
+tools, which in turn can support multiple languages with minimal effort.
+LSP is a win for both language providers and tooling vendors!
+
+Since our 4.0beta2 version, Spyder is now one of such development tools! We
+developed our own client to communicate with any server that implements the
+LSP v3.0 through a transport layer that uses ZeroMQ sockets. Code completion,
+signatures and docstring extraction were rewritten to take advantage of this
+architecture, and mouse hovers were added too (other LSP features, such as
+workspace functionality, will come in future betas).
+
+Our current support is geared towards Python, using the great
+[Python-Language-Server](https://github.com/palantir/python-language-server)
+package. This has allowed us to provide fine-grained configurability for
+[Pycodestyle](http://pycodestyle.pycqa.org/en/stable/) and
+[Pydocstyle](http://www.pydocstyle.org/en/stable/) options, and in future
+betas we’ll also add the ability to use and configure code formatters such
+as yapf and autopep8.
+
+![Hover hint and calltips](/images/spyder-hover-hint-calltip.png)
+
+We offer support to configure LSP servers  for other programming languages
+as well. In the future we hope to include out-of-the-box support for
+programming languages in the scientific computing spectrum, such as
+Fortran, Julia and C/C++.
+
+![Other languages](/images/spyder-preferences-lsp.png)
+
+## Plots pane
+
+Following the steps and inspiration of RStudio, another well known IDE in the
+Data Science space, Spyder now includes a Plots viewer pane. Since beta2 plots
+generated in every console will be displayed there, and the pane will be
+updated automatically when switching between consoles. Besides, users will
+have the ability to browse the history of all plots generated in a session.
+
+![Other languages](/images/spyder-plots.png)
+
+## And much more
+
+There are many more features that have been developed in the previous 10
+months! These include:
+
+1. **Autosave** files in the editor. With this Spyder will recover your
+unsaved files in case it crashes before you save them. 
+2. Dedicated **Sympy**, **Cython** and **Pylab** consoles. This will make it
+very simple to quickly explore and create code for these libraries.
+3. OS level **window pane undocking**. This will allow users to easily
+organize panes in different monitors.
+4. Support for **[multi-indexes](https://pandas.pydata.org/pandas-docs/stable/user_guide/advanced.html)**
+in our Dataframe viewer.
+
+We will describe all of these additional enhancements in greater detail in a
+future blog post.
+
+## How can you help?
+
+If you are a Spyder user and would like to help us test this beta release, you
+can do so by installing it with conda, from our beta releases channel:
+
+```bash
+$ conda update qt pyqt
+$ conda install spyder=4.0.0b2 --channel spyder-ide
+```
+
```

Comment:
Please add here the following

*This is a safe process because Spyder now uses a different configuration directory for its beta versions, so you can easily switch between our stable and beta releases without fearing to damage your current installation*.